### DISCUSSION TITLE
Parámetros de imágenes de Blogger

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#10] [Announcements]
# Parámetros de imágenes de Blogger

Google tiene el servidor de imágenes gratuito más poderoso y hoy te voy a enseñar a sacar el máximo provecho a las imágenes.

https://www.zkreations.com/2022/11/parametros-de-imagenes-de-blogger.html

**Comment by gianles** [comment]
gran explicación, creo que dejas claro muchas cosas sobre los parámetros de las imágenes que subimos directamente a blogger y lo que podemos jugar con ella, cambiando totalmente su resolución y tamaña e aspecto!

**Reply by danieIabel** [reply]
Asi es amigo, es increíble lo fácil que es optimizar y manipular cualquier imagen subida a Blogger. Ojalá en algún futuro los ingenieros de Blogger nos brinden a los desarrolladores más facilidades para implementar esto, pero aun así, sigue siendo muy poderoso.

**Comment by cristianime** [comment]
Daniel veo que usas el metodo de subir videos con blogger, es mejor que google drive?

**Reply by danieIabel** [reply]
Diría que si amigo, aunque no he hecho pruebas suficientes, pero en cuanto a reproducción al parecer el ancho de banda es ilimitado.

**Reply by cristianime** [reply]
Bro el parámetro a mi no me funciona, lo pongo y los videos no cargan, pero a ti si te salen, si publicas un artículo sobre como haces tú para que funcione ayudaría a mucha gente 😀

**Reply by danieIabel** [reply]
Cuando subes un video debes esperar a que se procese en los servidores de Google Fotos, este proceso puede durar muchas horas si el video es muy pesado y largo, es lo mismo que ocurre cuando subes un video a YouTube, es decir, que de primeras no estará procesado pero de a poco se irán procesando las distintas resoluciones. De todas formas haré un articulo amigo, gracias por la sugerencia.

**Reply by danieIabel** [reply]
Debido a cambios recientes en la plataforma (Blogger), ya no es posible subir videos a Blogger mediante el cargador de imágenes, ademas, los videos que ahí se encuentren, ya no estarán disponibles.

**Comment by llusan** [comment]
Simplemente GENIAL. Me sabía varias, pero las de compresión son reveladoramente nuevas para mí. GRACIAS por la información.

**Comment by llusan** [comment]
¿Será posible superar los 90 días de caché?
Lamentablemente, no me deja poner más allá de e90, ej. si pongo e100 lo reduce a e90.

Estoy intentando tener un buen puntaje en el PageSpeed, pero esto no me deja xd

**Reply by danieIabel** [reply]
Buenos dias amigo, tienes razon, acabo de probar y ahora solo acepta como máximo 90 dias. He actualizado la información de la entrada. Por otro lado, la cantidad que se recomienda mantener en caché un archivo, depende de cuánto cambia ese archivo en el tiempo. Por ejemplo, la imagen que quieres mantener mas de 90 dias, ¿Realmente no cambiará por ese periodo de tiempo? Yo generalmente suelo usar como maximo 30 dias ya que suelo actualizar mis portadas o hacer cambios sutiles, además no siempre se mantienen con el tiempo en la misma página.

**Reply by llusan** [reply]
Hola. 
En realidad, hoy estuve obsesionado con tener un buen puntaje en el PageSpeed de Google y me seguía arrojando en rojo aquellas imágenes con caché de 7 días, que luego cambié a 30, 60 y finalmente en 90. Por mi parte, lo dejaría con caché eterno xd, ya que las imágenes de blogger no son actualizables, se quedan tal cual están con su larga dirección. xd
No sé cuál es el tiempo ideal que solicita el PageSpeed, porque incluso puse la imagen en formato webp y en calidad V3 para que no exija tanto, y seguía saliéndome la alerta en rojo.

Ojalá que Blogger pueda habilitar más tiempo 😅, al menos quiero llegar a puntaje 70 teniendo ads xd

**Reply by danieIabel** [reply]
Si gustas comparte el link del test amigo para poder apreciar las recomendaciones que te da la herramienta y poder ayudarte en algo.
